kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/expressions/DefaultExpressionEvaluatorSuite.scala

```diff
@@ -388,9 +390,154 @@ class DefaultExpressionEvaluatorSuite extends AnyFunSuite with TestUtils {
     val right = literals(4)
     Seq.range(5, literals.length).foreach { idx =>
       comparatorToExpResults.foreach { comparator =>
-          testComparator(comparator, right, literals(idx), null)
+        testComparator(comparator, right, literals(idx), null)
+      }
+    }
+  }
+
+  test("evaluate expression: element_at") {
+    import scala.collection.JavaConverters._
+    val nullStr = null.asInstanceOf[String]
+    val testMapValues = Seq(
+      Map("k0" -> "v00", "k1" -> "v01", "k3" -> nullStr, nullStr -> "v04").asJava,
+      Map("k0" -> "v10", "k1" -> nullStr, "k3" -> "v13", nullStr -> "v14").asJava,
+      Map("k0" -> nullStr, "k1" -> "v21", "k3" -> "v23", nullStr -> "v24").asJava,
+      null
+    )
+    val testMapVector = new ColumnVector {
+      override def getDataType: DataType =
+        new MapType(StringType.INSTANCE, StringType.INSTANCE, true)
```

Comment:
can you comment what the true param is? `... StringType.INSTANCE, true /* foo */)`